com.microsoft.mrtk.graphicstools.unity/Runtime/Utilities/DrawFullscreenPass.cs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#if GT_USE_URP
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+namespace Microsoft.MixedReality.GraphicsTools
+{
+    /// <summary>
+    /// Draws full screen mesh using given material and pass and reading from source target.
+    /// Forked from: https://github.com/Unity-Technologies/UniversalRenderingExamples/tree/master/Assets/Scripts/Runtime/RenderPasses
+    /// </summary>
+    internal class DrawFullscreenPass : ScriptableRenderPass
+    {
+        public FilterMode filterMode { get; set; }
```

Comment:
nit: can you add [API reference](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-graphicstools/contributing#api-reference-documentation) for all public properties and methods?